### DISCUSSION TITLE
Allow deleting translations from translations interface

### DISCUSSION THREAD

**Discussion by FINDarkside** [directus/directus#16324] [Feature Requests]
One of the biggest UX problems we've stumbled while using Directus is that when a person inserting content accidentally inserts content in the wrong language they don't have any easy way to fix their mistake. They could open a split view and copy all the fields to the right language, but then they would also need to be able to delete the translation with wrong language which they cannot do at the moment. I know it's possible to delete them from the translations collection but making it visible and telling editors to go find and delete their translation there isn't really an option as it'd be way too error prone.

Right now our workflow is that when an editor inserts content in the wrong language they will just archive that content completely and insert everything again in the correct language. Would be great if the translation UI had some kind of button/icon or whatever to delete the translation. That would save a lot of time.

**Comment by benhaynes** [comment]
It seems as though there should be a "clear value" button for these relational fields too (like standard field have in the label's context menu).

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by taco-chen** [comment]
I've just met the same issue. Please re-open the discussion.

**Reply by FINDarkside** [reply]
This one is open: https://github.com/directus/directus/discussions/19895
